src/main/java/se/sundsvall/casemanagement/api/CaseStatusResource.java

```diff
@@ -41,56 +41,54 @@
 @ApiResponse(responseCode = "502", description = "Bad Gateway", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 public class CaseStatusResource {
 
-    private final ByggrService byggrService;
+	private final ByggrService byggrService;
 
-    private final EcosService ecosService;
+	private final EcosService ecosService;
 
-    private final CaseDataService caseDataService;
+	private final CaseDataService caseDataService;
 
-    private final CaseMappingService caseMappingService;
+	private final CaseMappingService caseMappingService;
 
-    public CaseStatusResource(ByggrService byggrService, EcosService ecosService, CaseDataService caseDataService, CaseMappingService caseMappingService) {
-        this.byggrService = byggrService;
-        this.ecosService = ecosService;
-        this.caseDataService = caseDataService;
-        this.caseMappingService = caseMappingService;
-    }
+	public CaseStatusResource(final ByggrService byggrService, final EcosService ecosService, final CaseDataService caseDataService, final CaseMappingService caseMappingService) {
+		this.byggrService = byggrService;
+		this.ecosService = ecosService;
+		this.caseDataService = caseDataService;
+		this.caseMappingService = caseMappingService;
+	}
 
-    @GetMapping(path = "organization/{organizationNumber}/cases/status", produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
-    @Operation(description = "Returns the latest status for each of the cases where the specified organization has the role \"applicant\".")
-    @ApiResponse(responseCode = "200", description = "OK - Successful operation")
-    public ResponseEntity<List<CaseStatusDTO>> getStatusByOrgNr(@Pattern(regexp = Constants.ORGNR_PATTERN_REGEX, message = Constants.ORGNR_PATTERN_MESSAGE)
-    @Schema(description = "Organization number with 10 or 12 digits.", example = "20220622-2396")
-    @PathVariable String organizationNumber) {
+	@GetMapping(path = "organization/{organizationNumber}/cases/status", produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
+	@Operation(description = "Returns the latest status for each of the cases where the specified organization has the role \"applicant\".")
+	@ApiResponse(responseCode = "200", description = "OK - Successful operation")
+	public ResponseEntity<List<CaseStatusDTO>> getStatusByOrgNr(@Pattern(regexp = Constants.ORGNR_PATTERN_REGEX, message = Constants.ORGNR_PATTERN_MESSAGE)
```

Comment:
Out-of-scope, but maybe just return a ResponseEntity.notFound instead of throwing an exception with NOT_FOUND status?

Reply:
I'm not entirely convinced we should return not found at all here. But I don't want to touch the API in this service at the momenent. We can't release a new version every week.